### PR TITLE
Small refactoring to better code readability

### PR DESCRIPTION
**Pull request by likemusic**
### Description
Small refactoring to better code readability

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: app/code/Magento/Backend/Block/Widget/Form/Element/Dependence.php [hunk 1]
```diff
@@ -124,14 +124,17 @@ protected function _toHtml()
...
-            $this->_getDependsJson() .
-            ($this->_configOptions ? ', ' .
-            $this->_jsonEncoder->encode(
-                $this->_configOptions
-            ) : '') . '); });</script>';
+
+        $dependsJson = $this->_getDependsJson();
+        $configOptions = $this->_configOptions ? $this->_jsonEncoder->encode($this->_configOptions) : null;
+        $configOptionsStr = $configOptions ? ', ' . $configOptions : '';
+        $paramsStr = $dependsJson . $configOptionsStr;
```
Comment (orlangur) [inline app/code/Magento/Backend/Block/Widget/Form/Element/Dependence.php]: IMO readability could be slightly improved if you do like this:
```
$params = $this->_getDependsJson();
if ($this->_configOptions) {
    $params .= ', ' . $this->_configOptions;
}
```

**Review by orlangur — CHANGES_REQUESTED** [review]

File: app/code/Magento/Backend/Block/Widget/Form/Element/Dependence.php (hunk 1 above)
Comment (likemusic) [reply app/code/Magento/Backend/Block/Widget/Form/Element/Dependence.php]: @orlangur, Absolutely agree with you. I updated pull request according to your suggestion.

**Review by orlangur — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @orlangur, thank you for the review.
[ENGCOM-1960](https://jira.corp.magento.com/browse/ENGCOM-1960) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
Hi @likemusic. Thank you for your contribution. 
Changes from your Pull Request will be available with the upcoming 2.2.6 release.

Please, consider to port this solution to 2.3 release line.
You may use [Porting tool](https://github.com/magento/magento2/wiki/Simple-way-to-port-your-solutions-automatically) to port commits automatically.

**Pull request merged**